examples/foreach/foreach.bal

```diff
@@ -7,24 +7,24 @@ public function main() {
     // The `foreach` statement can be used to iterate an array. Each iteration returns an element in the array. 
     // The index of the corresponding element is not returned.
     foreach var v in fruits {
-        io:println("fruit: " + v);
+        io:println("fruit: ", v);
     }
 
     io:println("\nIterating a map :");
     map<string> words = { a: "apple", b: "banana", c: "cherry" };
 
-    // Iterating a `map` will return the key (`string`) and the value as a `tuple` variable.
+    // Calling the `.entries()' method on a `map` will return the key (`string`) and the value as a `tuple` variable.
```

Comment:
Shall we change the example to iterate the map directly, instead of iterating over its entries? My concern is whether it might send the wrong idea that you have to iterate a map on its entries, instead of directly iterating over it.

Reply:
Examples iterating both with and without `.entries()` are listed in `maps` example. Do we still need to have the change here?

Reply:
Just above that code snippet, an example without the `.entries()` is shown.

Reply:
Added an example without `entries()` call just above the code snippet.